.ci/jenkins/config/branch.yaml

```diff
@@ -26,24 +26,7 @@ environments:
       QUARKUS_BRANCH: main
       DROOLS_BUILD_MVN_OPTS: -Dfull
     ids:
-    - quarkus
-  quarkus-3:
-    pull_request_default_check: false
-    env_vars:
-      BUILD_MAVEN_TOOL: kie-maven-3.9.1
-      BUILD_JDK_TOOL: kie-jdk17
-      BUILD_MVN_OPTS: -Denforcer.skip
-      # DROOLS_BUILD_MVN_OPTS: -Dfull #  "No enabled as it does not compile in quarkus-3
-      # BUILD_MVN_OPTS_QUARKUS_UPDATE: -Dfull
-    ids:
-    - quarkus
-    - quarkus3
-  quarkus-branch:
-    env_vars:
-      QUARKUS_BRANCH: '2.16'
-      DROOLS_BUILD_MVN_OPTS: -Dfull
-    ids:
-    - quarkus
+    - quarkus    
   quarkus-lts:
```

Comment:
I think quarkus-lts can be dropped too? As main branch from apache is already using Quarkus LTS

Reply:
Also one reminder is that that this removal of the .ci/environments/quarkus-3 and .ci/environments/quarkus-lts also need to be done for Kogito repositories in midstream.

Reply:
i didn't have time to follow on all repositories yet, but it is in my todo list